app/src/main/java/com/futurice/android/reservator/model/Model.java

```diff
@@ -0,0 +1,93 @@
+package com.futurice.android.reservator.model;
+
+import android.accounts.AccountManager;
+import android.content.ContentResolver;
+import android.os.Handler;
+import android.content.Context;
+import android.widget.Toast;
+
+import com.futurice.android.reservator.common.PreferenceManager;
+import com.futurice.android.reservator.model.platformcalendar.PlatformCalendarDataProxy;
+import com.futurice.android.reservator.model.platformcontacts.PlatformContactsAddressBook;
+
+public class Model {
```

Comment:
I think we can state the name better so that we know which fragment this Model is used for.

Reply:
Model class is meant to be the global Model for the whole app.